tests/integration/repos_test.go

```diff
@@ -145,3 +145,31 @@ func TestRepositories_EditBranches(t *testing.T) {
 		t.Fatalf("Repositories.Delete() returned error: %v", err)
 	}
 }
+
+func TestRepositories_List(t *testing.T) {
+	if !checkAuth("TestRepositories_List") {
+		return
+	}
+
+	_, _, err := client.Repositories.List("", nil)
+	if err != nil {
+		t.Fatalf("Repositories.List('') returned error: %v", err)
+	}
+
+	_, _, err = client.Repositories.List("google", nil)
+	if err != nil {
+		t.Fatalf("Repositories.List('google') returned error: %v", err)
+	}
+
+	opt := github.RepositoryListOptions{Sort: "created"}
+	repos, _, err := client.Repositories.List("google", &opt)
+	if err != nil {
+		t.Fatalf("Repositories.List('google') with Sort opt returned error: %v", err)
+	} else {
+		for i, repo := range repos {
+			if i > 0 && (*repos[i-1].CreatedAt).Time.Before((*repo.CreatedAt).Time) {
+				t.Fatalf("Repositories.List('google') with default descending Sort returned incorrect order")
+			}
+		}
```

Comment:
Minor style suggestion (completely optional). There's no need for the for loop to be inside the else clause, you can just pull it out to normal path.